script_runner/audit_log.py

```diff
@@ -38,3 +38,14 @@ def log(self, user: str, group: str, function: str, region: str) -> None:
                 "source_category": "infra-tools",
             },
         )
+
+
+class SlackEventLogger(AuditLogger):
+    def __init__(self, key: str, url: str) -> None:
+        self.__notifier = slack_notifier.SlackNotifier(key, url)
+
+    def log(self, user: str, group: str, function: str, region: str) -> None:
```

Comment:
i'm starting to wonder if it's better/less verbose to pass a list of regions here and log a single message since we send them all to the same place?